### DISCUSSION TITLE
uidmap with AD username format

### DISCUSSION THREAD

**Discussion by zuberua** [moby/moby#46057] [General]
Hi, we are using uidmap for our rootless docker installation (runnning this script: contrib/dockerd-rootless-setuptool.sh) for our AD users. The format of our AD user is DOMAIN\USERNAME.  As part of the dependencies check, the script is greping if the user has entry in the subuid and subgid file but the script is not able to detect out user entry due to the format of the username: DOMAIN\USERNAME.

The only way that we are able to get the subuid and gid requirements fullfilled for rootless docker is to have these 2 entries together:

DOMAINUSERNAME:100000:65536
DOMAIN\USERNAME:100000:65536

Can you kindly check u the scrip.

**Comment by corhere** [comment]
Duplicate of https://github.com/moby/moby/issues/44396
